events/uk/the_round.yaml

```diff
@@ -472,6 +472,18 @@ events:
     social: true
     organisation: The Round
 
+  - name: The Round Jane Austen Evening
+    links:
+      - "https://round.soc.srcf.net/diary2022-23"
+    start: 2023-03-1720:00:00+00:00
+    end: 2023-03-17T22:30:00+00:00
+    country: UK
+    city: Cambridge
+    styles: [regency, jane austen]
```

Comment:
Only the styles listed at https://github.com/qwandor/dancelist-data#dance-styles are supported currently.

Reply:
yes, this is ECD